client/components/NavBar/NavBarAvatarMenu.js

```diff
@@ -16,8 +20,7 @@ import styles from './nav-bar.css';
 const inLineStyles = {
   iconMenu: {
     iconStyle: { minWidth: 70, display: 'flex', flexDirection: 'row', alignItems: 'center' },
-    toggle: { label: { fontSize: '18px' }, thumbSwitched: { backgroundColor: 'red' } },
-    menuItem: { maxHeight: '30px', minHeight: '30px', lineHeight: '30px', width: '168px' },
+    toggle: { style: { top: '10px'}, label: { fontSize: '18px' }, thumbSwitched: { backgroundColor: 'red' } }
```

Comment:
please, can you add a trailing comma here?  And a space after  { top: '10px' ? 
Try to enable the lint option at .env, and the system will warn you about this simple styles issues.